tests/integrationtest/r/expression/builtin.result

```diff
@@ -713,7 +713,7 @@ create table tb5(a json, b bigint unsigned);
 insert into tb5 (a, b) values ('184467440737095516160', 18446744073709551615);
 select * from tb5 where cast(a as unsigned int)=b;
 a	b
-184467440737095500000	18446744073709551615
+1.844674407370955e20	18446744073709551615
```

Comment:
It's not a behavior change, but I found it's not compatible with MySQL :thinking:. Record in https://github.com/pingcap/tidb/issues/58901.